arc/studies/sc/files/arc-studies-2-1.scd

```diff
@@ -0,0 +1,26 @@
+(
+Server.default = Server.local;
+
+// MonomeArc accepts two arguments:
+// * rotation (0 is default)
+// * prefix (a string nickname for this arc, "/monome" is default)
+
+~a = MonomeArc.new(rotation: 0, prefix: "/monome");
```

Comment:
as rotation is a bonus/stretch feature, i'd generally suggest leaving it out of initializers to simplify the syntax.

i'd introduce it as an addendum at the end

Reply:
@tehn , makes sense!! you just mean in these examples, right? or do you think the method should be edited?

Reply:
@dndrks oh i just meant in the examples. i figure the API allows for leaving it off and then it passes the default of 0?